.github/workflows/pr-validation.yml

```
@@ -51,7 +56,7 @@ jobs:
         run: bun run typecheck
       
       - name: Run tests
-        run: bun test
+        run: npm test
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)